### PR TITLE
Add note about losing original values on save

### PR DESCRIPTION
**Pull request by jeremyharris**
Not entirely sure about the wording on this, let me know if you want me to change it.

Refs https://github.com/cakephp/cakephp/issues/9411

### CODE REVIEW

File: en/appendices/3-3-migration-guide.rst [hunk 1]
```diff
@@ -44,6 +44,9 @@ behavior that may effect your application:
 * The default ``ErrorController`` no longer disables ``Auth`` and ``Security``
   components, as it does not extend ``AppController``. If you are enabling these
   components through events, you will need to update your code.
+* ``Entity::clean`` now cleans original values, clearing them on save. This
+  behavior was a bug, as the entity's original state should not persist a
```
Comment (markstory) [inline en/appendices/3-3-migration-guide.rst]: 'original state should not be retained after a save'

Comment (jeremyharris) [inline en/appendices/3-3-migration-guide.rst]: :+1: Still include the bit below about the save reflecting the new state of the entity?

Comment (markstory) [inline en/appendices/3-3-migration-guide.rst]: Yeah.

**Comment by jeremyharris** [comment]
@markstory updated :)

**Pull request merged**